### ISSUE TITLE
"looting failed, possible softban..."  how to fiz it....

### ISSUE THREAD

**Issue by gordaogameplay** [NECROBOTIO/NecroBot#5333]
hello.

I'm a serious problem. when I start necrobot, I come across this message after a few minutes ......

--{
[01:22:57] Cubone      	- CP:  206  IV:  59.81%   [Best CP:   74  IV:  94.28%] (Candies: 27)
[01:23:02] (CatchSuccess) | (Normal) Zubat Lvl: 5 CP: (73/86) IV: 43.00% | Chance: 68.91% | 64.70m dist | with a PokeBall (106 left). | 210 EXP earned | Candies: 41 | lat: 34.014380 long: -118.494744
[01:23:04] Looking for incense Pokemon...
[01:23:05] Looking for Pokemon...
[01:23:05] Looking for incense Pokemon...
[01:23:06] Looking for Pokemon...
[01:23:07] Looking for incense Pokemon...
[01:23:11] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 1/45
[01:23:18] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 2/45
[01:23:25] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 3/45
[01:23:31] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 4/45
[01:23:37] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 5/45
[01:23:44] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 6/45
[01:23:49] Name: Broadway Theater INFO: Looting failed, possible softban. Unban in: 7/45
}--

how to fix it? or why does this happen?

thanks

**Comment by jinkawasaki333** [comment]
This has been asked thousands of time. Please read other issues before creating one.

**Comment by jamboljack** [comment]
just wait and see,,,, that the process until done...

**Comment by gordaogameplay** [comment]
and after seeing this message, I can not log in to account any on my computer, either the game or the necrobot. as if my ip was banned. then I restart my moden and back to normal.

**Comment by Lorden35519** [comment]
The game will attempt to softban you if you ave suspicious activity no matter what your ip is. What most likely happened is your modem wasn't off long enough to change your ip. Also, did it say successfully bypassed after those messages? If it didn't your softbanned and I would change your IP with a VPN before restarting the program.

**Comment by phparmand** [comment]
changing IP doesn't work. i have the same message on 50 different dedicated proxies

**Comment by Wasterl4nd** [comment]
I'm with the same issue, i restarted my modem 10 times and it doesn't work, anyone can help me ?

**Comment by Lorden35519** [comment]
You can't run more than one bot at the same time or you will get softbanned, you know that right?

**Comment by Lorden35519** [comment]
If changing your IP doesn't work and your running 1 bot only, you're probably hardbanned.

**Comment by Wasterl4nd** [comment]
Yes i only running one

**Comment by Wasterl4nd** [comment]
how i can remove hardban, if it's possible ?

**Comment by Lorden35519** [comment]
I don't know, it's never happened to me.

**Comment by Wasterl4nd** [comment]
:(, i'll wait then, thanks bro

**Comment by Lorden35519** [comment]
Wait ~24 hours then try and play on your phone. If it lets you loot PokeStops on your phone then I would run the bot again.

**Comment by Wasterl4nd** [comment]
Okay, i'll do this, if it works i say here

**Comment by Baleki** [comment]
i was running 4 bots yesterday but now get me ip ban

**Comment by Lorden35519** [comment]
Yup, when you run multiple bots at the same time you get IP banned. If you really want to run multiple you need to do one of two things. The first being use 1 bot for each computer, but if you only have one you need to do this second method. Inside Config.exe or whatever the hell it's called you can add a proxy. Proxy's usually cost money but there are a few free ones I believe just use Google.

**Comment by Linewalker** [comment]
https://github.com/NECROBOTIO/NecroBot/issues/5522#issuecomment-239429286
Please close. No code issue.
